tunnel/tunnel.go

```diff
@@ -42,6 +45,12 @@ type Session interface {
 	transport.Socket
 }
 
+var (
```

Comment:
Would be cool to have this on top of the src file like we talked about a while ago to be consistent with other source files and Go std library. Also, if we could add some comments here that would be dope.

Reply:
Done.